eng/scripts/Automation-Sdk-Init.ps1

```diff
@@ -39,6 +39,8 @@ $dotnetSdkVersion = $GlobalJson.sdk.version
 $installScript = GetDotNetInstallScript
     
 $dotnet = Join-Path $RepoRoot "../.dotnet"
+bash $installScript --install-dir $dotnet
+bash $installScript --install-dir $dotnet -c 3.1
```

Comment:
Why do we need .NET 3.1? That version is reaching EOL next month.

Reply:
Hello @weshaggard , netcoreapp3.1 is one of required target frameworks now.

Reply:
What requires .NET 3.1? Are we running tests? We shouldn't need to install .NET 3.1 to build a library and package.

Reply:
Current SDK generation pipeline does not run test for all required framework. We have updated our cadl-csharp emitter to remove dependency of .NET core 3.1 to use 6.0.  
But powerapps now cannot upgrade cadl-csharp emitter to new version. Because cadl-csharp emitter use 0.37.0 cadl compiler, while cadl-js emitter has not upgraded to 0.37.0, there will be pacakge conflict if powerapps upgrade cadl-csharp emitter.
After powerapps can upgrade cadl-csharp, I will update the scirpt not to install .Net core 3.1.

Reply:
@chunyu3 Thanks for the context please file a tracking issue to remove this dependency. I would still like to understand more about this dependency on .NET 3.1 from the cadl compiler and also would like to understand why the JS emitter would impact the .NET emitter in the PowerApps context. I will ask about this in tonight's sync meeting.